pootle/runner.py

```diff
@@ -124,13 +128,24 @@ def run_app(project, default_settings_path, settings_template,
                 print("File already exists, not overwriting.")
                 exit(2)
 
+        if args.db in ["mysql", "postgresql"]:
+            settings_template = "%s.%s" % (settings_template, args.db)
```

Comment:
Do we actually need to create two new sample settings files? I mean, if only the database setting is being altered why not replace it in the default template?